packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -47,12 +47,26 @@ abstract class ConstrainedLayoutBuilder<ConstraintType extends Constraints> exte
   /// The builder must not return null.
   final Widget Function(BuildContext context, ConstraintType constraints) builder;
 
+  /// Whether [builder] needs to be called again even if the layout constraints
+  /// are the same.
+  ///
+  /// When this widget is rebuilt, it is most likely that we need to call
+  /// [builder] to build this widget's child. However, subclasses may provide
+  /// ways in which the widget can be updated without needing to rebuild the
+  /// child. Such subclasses can use this method to tell the framework when the
+  /// child widget should be rebuilt.
+  @protected
```

Comment:
Maybe document explicitly that the _new_ widget is asked if it requires a rebuild and it is given the _old_ widget. Not the other way around.

Reply:
Done